### DISCUSSION TITLE
Guide for self hosting on dokku

### DISCUSSION THREAD

**Discussion by IgnisDa** [muety/wakapi#450] [Show and tell]
Hello. Wonderful project, and I am loving using it!

I use [Dokku](https://dokku.com) to self host these projects and was able to get Wakapi running there. I decided to document my steps and published them to my blog [here](https://blog.diptesh.me/self-hosting-wakatime).

That is all, and thanks for the project again!

**Comment by muety** [comment]
Thanks a lot for this!
